Javascript-2/Week-1/Movie App/style.css

```
@@ -0,0 +1,270 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Mulish:ital,wght@0,200..1000;1,200..1000&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+* {
+  font-family: "Montserrat", sans-serif;
+  background-color: rgb(33, 26, 26);
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  margin: 20px 0 20px 10px;
+  padding: 20px 0 0 50px;
+}
+section {
+    margin: 20px 0 20px 50px;
+  }
+  .site-name {
```

```suggestion
.site-name {
```
Mind the indentation here and for the other classes below

---

Hi,
First of all, thank you very much for your valuable advice. I have made the corrections to my code as per your suggestions. I hadn't realized there were so many mistakes, and your comments were incredibly helpful. Thanks to your guidance, I was able to resolve the issues.
Best regards,
Nusrath